db/migrate/20200213231744_remove_institution_id_from_school_certifying_officials.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveInstitutionIdFromSchoolCertifyingOfficials < ActiveRecord::Migration[5.2]
+  def change
+    safety_assured { remove_column :school_certifying_officials, :institution_id, :bigint }
```

Comment:
Why do you need `safety_assured`?

Reply:
When running `bundle exec rake db:migrate` without wrapping 

`remove_column :school_certifying_officials, :institution_id, :bigint`

I get a message regarding strong migrations 
`=== Dangerous operation detected #strong_migrations ===`
Which ends the migrations and and recommends wrapping this block with a `safety_assured { ... } block`, which allowed the migrations to be run successfully.

Reply:
Does it give more info?  why is it a dangerous operation?

Reply:
```
=== Dangerous operation detected #strong_migrations ===

ActiveRecord caches attributes which causes problems
when removing columns. Be sure to ignore the column:

class SchoolCertifyingOfficial < ApplicationRecord
  self.ignored_columns = ["institution_id"]
end

Deploy the code, then wrap this step in a safety_assured { ... } block.

class StrongMigrations::Checker < ActiveRecord::Migration[5.2]
  def change
    safety_assured { remove_column :school_certifying_officials, :institution_id, :bigint }
  end
end
```

So, it wants you to add
```ruby
self.ignored_columns = ["institution_id"]
```
to `SchoolCertifyingOfficial`, then deploy.  Then run the migration that drops the column.  Then you can remove 
```ruby
self.ignored_columns = ["institution_id"]
```

Reply:
I see, I will ignore the column for now and temporarily remove that migration.

Once deployed I can submit another PR to add the migration back in and remove `self.ignored_columns = ["institution_id"]`